calicoctl/commands/clientmgr/client.go

```diff
@@ -30,6 +33,37 @@ import (
 // parameter not loaded from file.
 func NewClient(cf string) (client.Interface, error) {
 	cfg, err := LoadClientConfig(cf)
+
+	// Set kubeconfig path
+	kubeconfigPath, exists := os.LookupEnv("KUBECONFIG")
+	if !exists {
+		kubeconfigPath = filepath.Join(
+			os.Getenv("HOME"), ".kube", "config",
+		)
+	}
+	cfg.Spec.KubeConfig.Kubeconfig = kubeconfigPath
+
+	// Look up datastore type
+	config, err := clientcmd.BuildConfigFromFlags("", kubeconfigPath)
+	if err != nil {
+		log.Info("Error loading config")
+		return nil, err
+	}
+
+	// If datastore type env var found, set it
+	datastoreType := apiconfig.EtcdV3
```

Comment:
Maybe we should default to "Kubernetes" unless "ETCD_ENDPOINTS" is specified in the config?